src/containers/check-page/CheckPage.module.css

```diff
@@ -1,3 +1,3 @@
 :global(#cognito-flow--root) {
-  z-index: 100;
+  z-index: 10001;
```

Comment:
can we set this in js instead and pull from the main zindex file? or maybe put it in the zindex file and reference the const with a comment here so other people know where in the hierarchy this could fall